File: src/test/java/upbrella/be/docs/utils/ApiDocumentUtils.java
package upbrella.be.docs.utils;

import org.springframework.restdocs.operation.preprocess.OperationRequestPreprocessor;
import org.springframework.restdocs.operation.preprocess.OperationResponsePreprocessor;

import static org.springframework.restdocs.operation.preprocess.Preprocessors.*;

public interface ApiDocumentUtils {
    static OperationRequestPreprocessor getDocumentRequest() {
        return preprocessRequest(
                modifyUris()
                        .scheme("https")
                        .host("api.upbrella.co.kr")
                        .removePort(),
                prettyPrint());
    }

    static OperationResponsePreprocessor getDocumentResponse() {
        return preprocessResponse(prettyPrint()); // (3)
    }
}

File: src/test/kotlin/upbrella/be/docs/utils/ApiDocumentUtils.kt
package upbrella.be.docs.utils

import org.springframework.restdocs.operation.preprocess.OperationRequestPreprocessor
import org.springframework.restdocs.operation.preprocess.OperationResponsePreprocessor
import org.springframework.restdocs.operation.preprocess.Preprocessors.*

object ApiDocumentUtils {
    @JvmStatic
    fun getDocumentRequest(): OperationRequestPreprocessor {
        return preprocessRequest(
            modifyUris()
                .scheme("https")
                .host("api.upbrella.co.kr")
                .removePort(),
            prettyPrint()
        )
    }

    @JvmStatic
    fun getDocumentResponse(): OperationResponsePreprocessor {
        return preprocessResponse(prettyPrint()) // (3)
    }
}